fortanix-sgx-tools/Cargo.toml

```diff
@@ -23,6 +23,11 @@ sgxs = { version = "0.6.0-rc1", path = "../sgxs" }
 sgx-isa = { version = "0.2.0-rc1", path = "../sgx-isa" }
 
 # External dependencies
-xmas-elf = "0.6.0" # Apache-2.0/MIT
-clap = "2.2.5"     # MIT
-failure = "0.1.1"  # MIT/Apache-2.0
+xmas-elf = "0.6.0"         # Apache-2.0/MIT
+clap = "2.2.5"             # MIT
+failure = "0.1.1"          # MIT/Apache-2.0
+failure_derive = "0.1.1"   # MIT/Apache-2.0
+serde_derive = "1.0.84"    # MIT/Apache-2.0
+serde = "1.0.84"           # MIT/Apache-2.0
+toml = "0.4.10"            # MIT/Apache-2.0
+num_cpus = "1.9.0"         # MIT/Apache-2.0
```

Comment:
Missing newline. Please configure your editor to automatically add a newline at the end of a file